src/index.js

```diff
@@ -1,6 +1,69 @@
 import renderRecentScore from './modules/recentScore.js';
 import renderAddScore from './modules/addScore.js';
 import './style.css';
+import { createGame, setScore, fetchScore } from './modules/data.js';
 
 renderRecentScore();
-renderAddScore();
\ No newline at end of file
+renderAddScore();
+
+const getId = (result) => {
+  const arr = result.split(' ');
+  return arr[3];
+};
+
+const gameID = createGame({ name: 'ghost recon' })
+  .then((data) => getId(data.result));
+
```

Comment:
- [ ] [Optional] Everything in the application works properly. However, it would be nice to use one `gameId` for your application. The reason I suggested this is that anytime I reload the page, all the scores are gone. This is because a new `gameId` is fetched anytime I reload the page. For a good user experience, it would be great to have one `gameId`. You can put instructions in the `README.md` file to guide users on how to generate a new `gameId` if a user wants to create a new game. 👍

Reply:
Fixed this issue

Reply:
@bobb-Rob, great work 👏